extension/app/manifest.json

```diff
@@ -23,7 +23,9 @@
 		"webNavigation"
 	],
 	"web_accessible_resources": ["vendor/*", "js/*", "inpage/*"],
-	"content_security_policy": "script-src 'self' 'sha256-Wt5bI4gKonDfzuTQo2amECh42QZMZy6vq93zZbd+q3M=' blob: 'unsafe-eval'; object-src 'self';",
+	// all hashes except the first one are from .html files `<script type = 'module'>import '../js/popup.js'</script>` parts
+	// this was done to workaround bug: https://bugzilla.mozilla.org/show_bug.cgi?id=1803984
+	"content_security_policy": "script-src 'self' 'sha256-1yZWjAKHObRCaBauCTkoZtWkihs/4uwsiu6Hnx3OkTs=' 'sha256-Uit0MJakCHko/a4ufXGIQJssrw/9TvLamXN+6ltsoG8=' 'sha256-H3mVX1ZCLoCDyvKnpCmo127wWUkke4ADcFzg0RfP2ZM=' 'sha256-8HfgBeyLUz3lfmitlAsCqVcTegmlIKg2JkhG9r1SC3o=' 'sha256-0TgMBbiL41Lbex4DiXR11hCqokTcIKXw6/HJa+UJ7H4=' 'sha256-X9JOx19AgjNibvT805ows7My7i7Soq9cP0ELHgAiHKg=' blob: 'unsafe-eval'; object-src 'self';",
```

Comment:
What is the first hash then, and why do we need it?

Reply:
Also, there are 6 HTML files and 6 hashes, so is one missing?

Reply:
background.html does not need preact (it has no interface) and I didn't modify it. The first hash is the hash of this:
```ts
<script type = 'importmap'>
		{
			"imports": {
				"ethers": "../vendor/ethers/ethers.esm.js",
				"webextension-polyfill": "../vendor/webextension-polyfill/browser-polyfill.js",
				"preact": "../vendor/preact/preact.module.js",
				"preact/jsx-runtime": "../vendor/preact/jsx-runtime/jsxRuntime.module.js",
				"preact/hooks": "../vendor/preact/hooks/hooks.module.js",
				"funtypes": "../vendor/funtypes/index.mjs",
				"node-fetch": "../vendor/node-fetch/index.mjs",
				"@zoltu/ethereum-abi-encoder": "../vendor/@zoltu/ethereum-abi-encoder/index.js",
				"@zoltu/ethereum-crypto": "../vendor/@zoltu/ethereum-crypto/index.js",
				"@zoltu/rlp-encoder": "../vendor/@zoltu/rlp-encoder/index.js",
				"@darkflorist/address-metadata": "../vendor/@darkflorist/address-metadata/index.js"
			}
		}
</script>
```

Reply:
The problem isn't specific to `preact`, it is a problem with *all* named imports (e.g., `ethers`)

Reply:
It may be a good idea to add a comment either above the CSP line or above this line indicating the hash.  Trying to guess where a hash is from is next to impossible.  😖

Reply:
> The problem isn't specific to `preact`, it is a problem with _all_ named imports (e.g., `ethers`)

Wonder why I am not getting any errors from Ethers at all for background page